### PR TITLE
Integration tests for copr build

### PR DESCRIPTION
**Pull request by rpitonak**
TODO
- [x] Use openshift template for the pod in OpenShift (done by @TomasTomecek in #171)
- [x] Utilize requre (cc @jscotka )
- [x] Green CI

### CODE REVIEW

File: tests/integration/test_copr.py [hunk 1]
```diff
@@ -1,81 +1,76 @@
...
-    flexmock(PackitAPI).should_receive("watch_copr_build").and_return("failed").once()
 
-    jobs.SteveJobs().process_message(pr_event)
+def test_submit_copr_build_pr_event(pr_event):
+    steve = SteveJobs()
+    result = steve.process_message(pr_event)
 
+    assert result
+    assert "copr_build" in result["jobs"]
+    assert result["jobs"]["copr_build"] == "success"
```
Comment (TomasTomecek) [inline tests/integration/test_copr.py]: so simple!

**Comment by rpitonak** [comment]
@jscotka tests are passing locally for me with the stg credentials. How to try:
```
1. $ git clone https://gitlab.cee.redhat.com/user-cont/packit-service
2. $ checkout to this PR
3. $ make tests-inside-openshift PATH_TO_SECRETS=../packit-service/secrets
```

Let me know if something will not work for you.

**Comment by rpitonak** [comment]
recheck

**Comment by rpitonak** [comment]
recheck

File: files/test-in-openshift-secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+---
+- name: Deploy secrets for integration tests
+  hosts: localhost
+  vars:
+    path_to_secrets: "../secrets"
+    deployment_dir: "/tmp/deployment"
+    deployment: "stg"
```
Comment (TomasTomecek) [inline files/test-in-openshift-secrets.yaml]: deployment and path_to_secrets vars are not used

**Comment by jscotka** [comment]
work continues in
https://github.com/packit-service/packit-service/pull/199

File: files/test-in-openshift-secrets.yaml (hunk 2 above)
Comment (jscotka) [reply files/test-in-openshift-secrets.yaml]: these variables seems to be very important, I've removed them and it start to failing, somehow used inside ``includes``

**Comment by jscotka** [comment]
closing this PR, as work continues in #199

**Pull request closed without merging**